documentation/src/gate_operations/intro.md

```diff
@@ -1,13 +1,13 @@
 # List of Gate Operations
 
-Operations are the atomic instructions in any quantum program that can be represented by qoqo/roqoqo. Gate operations are single-, two- or multi-qubit unitary operations that apply a unitary transformation and can be executed on a any universal quantum computer. Mathematically, a gate can be represented by a unitary matrix.
+Operations are the atomic instructions in any quantum program that can be represented by qoqo/roqoqo. Gate operations are single-, two- or multi-qubit unitary operations that apply a unitary transformation and can be executed on a universal quantum computer. Mathematically, a gate can be represented by a unitary matrix.
```

Comment:
Would there be a better formulation to stress that these operations can always be executed on universal quantum computers? In the sense that if there is a universal quantum computer it must be able to execute that operation (potentially as a sequence of native operations)